PerimeterXModule/Internals/PxContext.cs

```diff
@@ -51,6 +51,8 @@ public class PxContext
 		public string VidSource { get; set; }
 		public string Pxhd { get; set; }
         public bool MonitorRequest { get; set; }
+		public LoginCredentialsFields LoginCredentialsFields { get; set; }
```

Comment:
fix indentation

Reply:
Everything looks fine in the source file (and not on Github)